pallets/funding/src/tests/2_evaluation.rs

```diff
@@ -76,6 +76,158 @@ mod round_flow {
 			let project_status = inst.get_project_details(project_id).status;
 			assert_eq!(project_status, ProjectStatus::FundingFailed);
 		}
+
+		#[test]
```

Comment:
In such a long test can we better define the different stages of the test for clarity? Or if possible split the test up in multiple tests. Either one is good for me.